### ISSUE TITLE
SKID: some guy idk

### ISSUE THREAD

**Issue by fowntain** [Minesraft2/Blooket-Cheats#703]
### Skid

https://github.com/OGBlooketCheater/Blooket-Cheaters-Menu

### Stolen

GUIs and yknow what’s funny? He put “no copying my code” in the readme LOL
Oh and also he made a fucking YouTube video about it in the tutorial file

**Comment by Madmase555** [comment]
BRUH

**Comment by Madmase555** [comment]
this man is pure evil

**Comment by Madmase555** [comment]
![2023-03-26_085317](https://user-images.githubusercontent.com/120183925/227777019-ac4afc8f-1125-424c-ba12-ac7a8be8179f.jpg)
caught him in 4k

**Comment by orangemonkey01** [comment]
go to issues on his older one I already told him to take it down

**Comment by OGBlooketCheater** [comment]
https://github.com/OGBlooketCheater/Blooket-Cheaters-Menu/blob/main/DISCLAIMER!

**Comment by OGBlooketCheater** [comment]
i have a disclaimer bro

**Comment by Minesraft2** [comment]
> ![Screen Shot 2023-03-30 at 8 58 21 AM](https://user-images.githubusercontent.com/124519462/228860313-90ef6aa0-45cf-46ea-8ac5-355c9c559ee9.png)

but I didn't get banned

**Comment by OGBlooketCheater** [comment]
wait if u didnt get banned what happened

**Comment by OGBlooketCheater** [comment]
Oh, wait why tho?

**Comment by OGBlooketCheater** [comment]
> go to issues on his older one I already told him to take it down

No.? Nothing is there

**Comment by OGBlooketCheater** [comment]
P.S the older one is a very old version of the mod menu

**Comment by OGBlooketCheater** [comment]
> ### Skid
> https://github.com/OGBlooketCheater/Blooket-Cheaters-Menu
> 
> ### Stolen
> GUIs and yknow what’s funny? He put “no copying my code” in the readme LOL Oh and also he made a fucking YouTube video about it in the tutorial file

I never said it was MY code. all i said was dont copy the code

**Comment by fowntain** [comment]
> > ### Skid
> > https://github.com/OGBlooketCheater/Blooket-Cheaters-Menu
> > ### Stolen
> > GUIs and yknow what’s funny? He put “no copying my code” in the readme LOL Oh and also he made a fucking YouTube video about it in the tutorial file
> 
> I never said it was MY code. all i said was dont copy the code

yeah yeah sure. it seems you were trying to hide the credit in a file instead of just putting it in the readme.
